tldr/cross_file_calls/parsers/go.py

```
@@ -0,0 +1,246 @@
+import os
+import re
+from typing import Dict, List, Optional
+
+from tldr.cross_file_calls.parsers.base import BaseParser
+from tldr.cross_file_calls.core import HAS_GO_PARSER, _get_go_parser
+
+
+class GoParser(BaseParser):
+    """Parser for Go files."""
+    
+    def extract_calls(self, file_path: str, timeout: Optional[float] = None) -> List[Dict]:
```

Ruff (0.14.11)

Unused method argument: `timeout`

(ARG002)